src/MICore/MICore.csproj

```diff
@@ -126,6 +126,10 @@
   <ItemGroup>
     <None Include="LaunchOptions.xsd" />
     <None Include="SetMIDebugLogging.cmd" />
+    <ContentWithTargetPath Include="osxlaunchhelper.scpt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <TargetPath>osxlaunchhelper.scpt</TargetPath>
+    </ContentWithTargetPath>
```

Comment:
This the reason why the script is being launched with `/usr/bin/osascript`, so that we don't need to worry about permissions.